tests/ltp/patches/fix_mkdir_mkdir02.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/testcases/kernel/syscalls/mkdir/mkdir02.c b/testcases/kernel/syscalls/mkdir/mkdir02.c
+index f09e3c33d..50dbcd2e5 100644
+--- a/testcases/kernel/syscalls/mkdir/mkdir02.c
++++ b/testcases/kernel/syscalls/mkdir/mkdir02.c
+@@ -17,6 +17,8 @@
+ #include <unistd.h>
+ #include <stdlib.h>
+ #include "tst_test.h"
++#include <sys/prctl.h>
++#include <linux/securebits.h>
+ 
+ #define TESTDIR1	"testdir1"
+ #define TESTDIR2	"testdir1/testdir2"
+@@ -54,10 +56,8 @@ static void verify_mkdir(void)
+ 		if (!fail)
+ 			tst_res(TPASS, "New dir inherited GID and S_ISGID");
+ 
+-		exit(0);
```

Comment:
Why was this removed?

Reply:
FORK_OR_VFORK in LTP returns always 0 in sgxlkl environment because sgx-lkl support single process. Original test case create a child  and exit from child. But in sgxlkl this will cause exit from the test program before test completion.  Updated the details in patch description.